spec/models/course_spec.rb

```diff
@@ -1624,4 +1624,17 @@
       )
     end
   end
+
+  describe "#is_unique_on_provider?" do
+    it "Returns the subjects using the correct service" do
```

Comment:
Sorry, I know this is nit-picking, but does it make sense to say it "Returns the subjects" when we're not testing the return value, and doesn't fit with the name of the method? Maybe it could just be "delegates to the correct service".